ui/components/ui/typography/typography.js

```diff
@@ -34,7 +33,6 @@ export default function Typography({
     {
       [`typography--align-${align}`]: Boolean(align),
       [`typography--color-${color}`]: Boolean(color),
-      [`typography--size-${fontSize}`]: Boolean(fontSize),
```

Comment:
In previous PRs I have added font-size to typography, but I found this should not have been there thus I removed it in favour of using variants H1-H9 for font-size.